tasks/bump.js

```diff
@@ -30,6 +33,20 @@ module.exports = function(grunt) {
       versionType: false
     });
 
+    // check if cwd exists
+    if (!fs.existsSync(opts.cwd)) {
+      grunt.fatal('The in "cwd" defined directory does not exist');
+    }
+
+    // set cwd as the working directory for exec
+    var execOpts = {
+      cwd: path.resolve(opts.cwd)
+    };
+
+    // set grunts default working directory
+    grunt.verbose.writeln('Changing working directory: ' + opts.cwd);
```

Comment:
I think it doesn't hurt if the current working directory is shown to the user. This is only in `--verbose` mode. But I'm also ok, with the change.